app/plugin_install.go

```diff
@@ -148,12 +148,16 @@ func (a *App) removePlugin(id string) *model.AppError {
 
 	// Remove bundle from the file store.
 	storePluginFileName := filepath.Join("./plugins", manifest.Id) + ".tar.gz"
-	if bundleExist, fileErr := a.FileExists(storePluginFileName); bundleExist && fileErr == nil {
+	bundleExist, fileErr := a.FileExists(storePluginFileName)
+
```

Comment:
nit: this _can_ be removed, usually the err checks are close to the call you're invoking :)

Reply:
My preference is always padding code 🤐🙈

Fixing for consistency.